BrainPortal/app/views/nh_invitations/_message_details.html.erb

```diff
@@ -0,0 +1,52 @@
+
+<%-
+#
+# CBRAIN Project
+#
+# Copyright (C) 2008-2012
+# The Royal Institution for the Advancement of Learning
+# McGill University
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.  
+#
+-%>
+<% messtype = mess.message_type # disguise confirmation for a flash notice until better styling added
+   prefix = "flash"
+   messclass = "#{prefix}_#{messtype}"
+%>
+
+<% unless mess.description.blank? %>
```

Comment:
Change the `unless something.blank?` to `if something.present?` it looks much nicer and avoids double negatives.

Reply:
ok Not sure is being blank something negative, `if` is shorter and as I know many devs consider `unless` bad style unless it removes a double negation.